lib/ansible/modules/cloud/azure/azure_rm_devtestlabarmtemplate_facts.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, (@zikalino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabarmtemplate_facts
+version_added: "2.8"
+short_description: Get Azure DevTest Lab ARM Template facts.
+description:
+    - Get facts of Azure DevTest Lab ARM Template.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    lab_name:
+        description:
+            - The name of the lab.
+        required: True
+    artifact_source_name:
```

Comment:
why it need artifact_source_name? is this copied from artifact source facts module?

Reply:
it's subresourcece of artifact_source.

Reply:
i return artifacts_source_name just for consistency.
somebody may query ARM templates from 2 artifact sources and then combine the lists for instance.